Source/JavaScriptCore/wasm/WasmModule.cpp

```diff
@@ -91,10 +121,17 @@ Ref<CalleeGroup> Module::getOrCreateCalleeGroup(VM& vm, MemoryMode mode)
     // https://bugs.webkit.org/show_bug.cgi?id=170607
     if (!calleeGroup || (calleeGroup->compilationFinished() && !calleeGroup->runnable())) {
         RefPtr<LLIntCallees> llintCallees = nullptr;
-        if (Options::useWasmLLInt())
-            llintCallees = m_llintCallees.copyRef();
-        calleeGroup = CalleeGroup::create(vm, mode, const_cast<ModuleInformation&>(moduleInformation()), WTFMove(llintCallees));
-        m_calleeGroups[static_cast<uint8_t>(mode)] = calleeGroup;
+        if (UNLIKELY(Options::useWasmIPInt())) {
+            RefPtr<IPIntCallees> ipintCallees = nullptr;
```

Comment:
nit: one line?

Reply:
Updated! Much cleaner now for all the cases.